test/puppetlabs/kitchensink/core_test.clj

```diff
@@ -108,6 +108,17 @@
        ["foo" "bar" "baz"] "foo, bar, and baz"
        ["foo" "bar" "baz" "qux"] "foo, bar, baz, and qux"))
 
+(deftest key->str-test
```

Comment:
Should we test the behavior of an argument that is neither a string nor a keyword?